cryptodoc/src/08_signatures.rst

```diff
@@ -353,7 +355,7 @@ The signature verification algorithm works as follows:
 
    **Input:**
 
-   -  ``m``: message bytes
+   -  ``m``: raw bytes to verify (the hash-code ``H``, must be already truncated if needed)
    -  EC_Publickey: ``Q``, domain(curve parameters(first coefficient a,
```

Comment:
The rendering is not very nice here, maybe add a few spaces and maybe also use the same notation as in ECDSA, where, for instance ``G`` is used for the base point. Same above. Moreover, but this is the notation used throughout the documentation: I think to use G as notation for a point (the basepoint) is really strange (why not use P?).

Reply:
Fixed it, also for `EC_Privatekey` and in ECGDSA.